charts/argo-workflows/values.yaml

```diff
@@ -305,6 +305,9 @@ controller:
   # -- Configure Argo Server to show custom [links]
   ## Ref: https://argoproj.github.io/argo-workflows/links/
   links: []
+  # -- Configure Argo Server to show custom [columns]
+  ## Ref: https://github.com/argoproj/argo-workflows/pull/10693
```

Comment:
nit: https://argoproj.github.io/argo-workflows/links/ would be better as a guide, instead of PR ref? 🙋

Reply:
links is a different feature than columns. unfortunately there's nothing in the docs about columns, so the best I could do was link to the PR that implemented the feature

Reply:
I thought it was about part of links because PR has its ref... 😇 
Thank you for explaining.

Reply:
yeah, it was a bit confusing - it's highlighted where I made a mistake, so all is good!